scraping.py

```diff
@@ -1,15 +1,18 @@
 import json
 import pandas as pd
 from selenium import webdriver
+#import geckodriver_autoinstaller
 
+
+#geckodriver_autoinstaller.install() 
```

Comment:
Remove these comments if they are unnecessary.

Reply:
These are needed for new user and he needed this line. But Pranav needs to check something about this.